flink/templates/ingress.yaml

```diff
@@ -1,5 +1,8 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "flink.fullname" . -}}
+{{- $serviceName := printf "%s-%s" (include "flink.fullname" .) "jobmanager-rest" }}
+{{- $servicePort := .Values.jobmanager.ports.ui -}}
+{{- $paths := .Values.ingress.path -}}
```

Comment:
This file was created with `helm create` and it uses the next input data format:
```ingress:
  enabled: true
  annotations:
    my_annotations: test+test2
  path: /
  hosts:
    - host: test.my-rf
      paths: [/]
    - host: test2.my-rf2
      paths: [/]
  tls: []```

I agree with updating hardcoded serviceName/servicePort but I think it's better to keep "default" format because it provides an ability to define separate paths for each host.